src/idpyoidc/server/session/claims.py

```diff
@@ -113,6 +113,9 @@ def get_claims_from_request(
             )
         else:
             _claims_by_scope = module.kwargs.get("add_claims_by_scope")
+            if not _claims_by_scope:
```

Comment:
should we do 

```suggestion
            if _claims_by_scope is not None:
```

to differentiate cases like `add_claims_by_scope: []`?

Reply:
The value of `add_claims_by_scope` is supposed to be a boolean.